### ISSUE TITLE
SegmentComparison module already provides comparison metrics for segments

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/slicer-manual-annotation#19]
I know that @maxradx was working on implementing Dice/Hausdorff to compare segmentations. 

There is already an extension to do this, so I'm mentioning it here so that people are aware: https://www.slicer.org/wiki/Documentation/Nightly/Modules/SegmentComparison

**Comment by maxradx** [comment]
Yes exact! It only takes a few lines of code. I am sure this already exists. In my code, I used sitk image https://simpleitk.org/SimpleITK-Notebooks/01_Image_Basics.html

**Comment by maxradx** [comment]
To discuss about relevance of including such functionalities. This is not urgent, but should be addressed (whether we go in or not) before a release. Note that a review of current existing module and extension in [issue 65](https://github.com/neuropoly/slicer-manual-annotation/issues/65) create a picture where actual module are quite limited in their functionality and not user-friendly to make it work.
